src/store/slicePopularCategory/selectorsPopularCategories.ts

```diff
@@ -0,0 +1,18 @@
+import { LOAD_STATUSES } from "../../constants";
+import { rootStore } from "../store";
+import { State } from './slicePopularCategories'
+
+export const getPopularCategoriesSlice = (state: rootStore): State => state.popularCategories;
+export const getLoadStatusSlice = (state: rootStore): LOAD_STATUSES =>
+  getPopularCategoriesSlice(state).loadStatus;
+  export const getPopularCategories = (state: rootStore) => getPopularCategoriesSlice(state).popularCategories;
```

Comment:
чтобы было меньше мест для потенциальных ошибок мы типизируем возвращаемое значение селектором, чтобы ts проверил что мы собирались вернуть (для этого и нужна типизация возвращаемого значения) и что реально вернули из фунции

import {Category } from "api";
```suggestion
  export const getPopularCategories = (state: rootStore): Category [] => getPopularCategoriesSlice(state).popularCategories;
```